node-fs-writefile/write-rand.js

```diff
@@ -0,0 +1,9 @@
+
+var fs = require('fs');
+
+var number = Math.random();
+var data = number.toString() + '\n';
+
+fs.writeFile('random.txt', Buffer.from(data, 'utf8'), (err) => {
```

Comment:
The `writeFile` method also accepts strings, so you don't _need_ to convert it to a `Buffer` first: https://nodejs.org/api/fs.html#fswritefilefile-data-options-callback